### ISSUE TITLE
Add: food label model estimator

### ISSUE THREAD

**Issue by ilyashusterman** [ilyashusterman/RecipeMealCalculator#1]
use :
1.https://github.com/Kemanth/Food-Classification-from-Images-Using-Convolutional-Neural-Networks-in-Keras-using-Tensorflow
2.https://github.com/stratospark/food-101-keras
3.https://github.com/danielvarga/keras-finetuning
